docs/UserGuide.md

```diff
@@ -124,7 +142,9 @@ Examples:
 
 ### Adding a label to a tag : `label`
 
-Adds one or more label to an existing tag. Duplicated labels will only be added once. 
+Adds one or more label to an existing tag.<br>
+You can use label to categorise tags.
+Warning: Duplicated labels will only be added once. 
 
```

Comment:
Can the Warning be in the information source thingy? (sorry i don't know what is the component called, but it was something like a box of)

Reply:
Done!